mbed_lstools/lstools_linux_generic.py

```diff
@@ -111,7 +113,12 @@ def list_mbeds(self):
                     mbed['target_id'] = mbed_htm_target_id
             mbed['target_id_usb_id'] = tid
             mbed['target_id_mbed_htm'] = mbed_htm_target_id
-            result.append(mbed)
+            if mbed['mount_point'] != None or self.list_unmounted:
```

Comment:
Could you change this check to:
```python
if mbed['mount_point'] or self.list_unmounted:
```

I think that's more idiomatic python. It should also catch some odd corner cases like the empty string as a mount point.